client/shared/src/search/query/filters.ts

```diff
@@ -176,6 +177,24 @@ export const LANGUAGES: string[] = [
     'Zig',
 ]
 
+const SOURCEGRAPH_DOT_COM_REPO_COMPLETION: Completion[] = [
+    {
+        label: 'Search a GitHub organization',
+        insertText: 'github\\.com/${1:ORGANIZATION}/.*',
```

Comment:
maybe for consistency?
```suggestion
        insertText: '^github\\.com/${1:ORGANIZATION}/.*',
```